src/tools/ImageConverter.cpp

```diff
@@ -0,0 +1,21 @@
+// Created by Maxandre Ogeret.
+// (c) 2022 University of Tartu - Autonomous Driving Lab.
+
+#include "tools/ImageConverter.h"
+
+ImageConverter::ImageConverter(DriveworksApiWrapper* driveworksApiWrapper, uint32_t width, uint32_t height,
+                               dwImageType outputImageType, dwImageFormat outputImageFormat,
+                               dwImageMemoryType outputImageMemoryLayout)
+  : driveworksApiWrapper_(driveworksApiWrapper)
+{
+  imageProperties_.type = outputImageType;
```

Comment:
imageProperties_ seems to be assigned, but never used?

Reply:
Indeed it was not used. I also totally removed this class.